Makefile

```diff
@@ -55,7 +65,7 @@ endif
 .PHONY: govet
 govet:
 ifneq ($(HAS_SERVER),)
-	@echo Running govet
+	@echo Running govet. (This is deprecated. Use make golangci-lint)
```

Comment:
Why not just remove it?

Reply:
For compatibility reasons ? If someone is used to running `make govet` or `make gofmt`, this gives them some time to migrate to new commands. We did the same for the main repo.

Reply:
I see the need for migration time on the main repo as there are >100 developers. But given that this is small repo, I would just do a clean cut. 1/5

Reply:
Ok, I will also make an announcement in the ~toolkit channel once this gets merged.

Reply:
Maybe even ~Developers.